PSix/EventCell.swift

```diff
@@ -32,7 +32,12 @@ class EventCell: UITableViewCell {
                 eventDateMonthLabel.text = event.startTime?.monthShortName.uppercaseString
                 eventDateDayLabel.text = event.startTime?.dayDoubleDigit
                 eventDateDayAndTime.text = EventPresenter.getDayHourOfStartConsideringWidth(event, boxWidth: eventDateDayAndTime.bounds.width, font: eventDateDayAndTime.font)
-                paymentCollectionStatus.text = EventPresenter.getPaymentStatus(event)
+                let paymentCollectionStatus = EventPresenter.getPaymentStatus(event)
```

Comment:
I don't like the fact that you have a presenter which isn't really acting like a presenter, more like a set of utility methods.